package.json

```diff
@@ -5,6 +5,10 @@
   "repository": "https://github.com/FormidableLabs/aws-lambda-serverless-reference",
   "author": "Ryan Roemer <ryan.roemer@formidable.com>",
   "license": "MIT",
+  "engines": {    
+    "node": "~12.14.0"  
```

Comment:
This reference project _could_ be installed on lower nodes that are still supported by AWS. Let's remove the restriction and `engine-strict` maybe?

Reply:
node 8 is EOL on lambda next week.  Can we restrict to > 10?

Reply:
This is an upstream app. I'd rather just leave it to `.nvmrc` than placing an enforcement here. If this was a lib and we knew we only had node10 code, then sure. But I don't see good value in an arbitrarily high enforcement...

Reply:
I'm resolving this convo and picking it up on the actual living code.